docs/open/open.md

```diff
@@ -0,0 +1,94 @@
+---
+id: open
+title: The "Open" Landscape
+custom_edit_url: null
+---
+
+Open Source with SLU is a champion and resource for all things "open" at Saint Louis University. Open science, open scholarship, open data, and open source software are closely interrelated concepts that collectively aim to promote transparency, accessibility, and collaboration in research, education, and knowledge dissemination. While each concept has its unique focus, they share common goals and principles, and their implementation reinforces one another.
+
+## Open Source Software
+
+Open source software enables collaborative research and sustainable innovation.
+
+[Open source software is based on a set of principles and practices](https://aws.amazon.com/what-is/open-source/). The most obvious practice is making the human-readable "source" code for software freely available for use, study, modification, and distribution. This fosters transparency, collaboration, and continuous improvement. Open source principles promote transparency by making code openly accessible, aligning with reproducibility goals in research. Open source solutions provide cost-effective, customizable alternatives to proprietary software, promoting vendor independence and long-term sustainability. The collaborative nature of open source communities drives sustained innovation and robust software development through diverse contributions and knowledge sharing.
```

Comment:
@yashb196 it would be better if we could point to a neutral third party source. I would be okay with just pointing to the OSI definition of open source (https://opensource.org/osd). We can also consider pointing to that definition for "principles" and some other document for "practices"